pages.ja/common/ls.md

```diff
@@ -0,0 +1,32 @@
+# ls
+
+> ディレクトリの内容を一覧表示します。
+> 詳しくはこちら: <https://www.gnu.org/software/coreutils/ls>.
+
+- ファイルを1行ごとに一覧表示:
+
+`ls -1`
+
+- 隠しファイルを含むすべてのファイルを一覧表示:
+
+`ls -a`
+
+- すべてのファイルを一覧表示し、ディレクトリ名の最後に `/` を付加する:
+
+`ls -F`
+
+- 全ファイルを長い形式（パーミッション、所有者、サイズ、修正日）で一覧表示します:
+
+`ls -la`
+
+- サイズを人間が読みやすい単位（KiB、MiB、GiB）で表示した長い形式での一覧表示:
+
+`ls -lh`
+
+- サイズ順（降順）に並べた長い形式での一覧表示:
+
+`ls -lS`
+
+- すべてのファイルの長い形式でのリストで、更新日が古いものから順に表示されます。:
```

Comment:
A period and a colon here?
```suggestion
- すべてのファイルの長い形式でのリストで、更新日が古いものから順に表示されます:
```

Reply:
No that is my mistake.
I appreciate you pointing it out.